libraries/adb-backend-webusb/src/backend.ts

```diff
@@ -101,9 +101,9 @@ export class AdbWebUsbBackend implements AdbBackend {
         return devices.map(device => new AdbWebUsbBackend(device));
     }
 
-    public static async requestDevice(): Promise<AdbWebUsbBackend | undefined> {
+    public static async requestDevice(filters: USBDeviceFilter[]): Promise<AdbWebUsbBackend | undefined> {
```

Comment:
Add a default value to maintain backward compatibility.

Reply:
Done 👍